css/src/components/message.scss

```diff
@@ -0,0 +1,52 @@
+.message {
+	display: grid;
+	grid-template:
+		'. time'
+		'avatar main';
+	grid-template-columns: max-content 1fr;
+	gap: $layout-1;
+
+	.message-time {
+		grid-area: time;
+		font-size: $font-size-9;
+	}
+
+	.message-avatar {
+		grid-area: avatar;
+		margin-block-start: $spacer-3;
+	}
+
+	.message-content {
+		grid-area: main;
+		padding: $spacer-5;
+		border: $border-width solid $border;
+		border-radius: $border-radius-lg;
+		box-shadow: $box-shadow-light;
+
+		.message-content-head {
```

Comment:
Would `message-content-header` be a more recognizable name?

Reply:
if you havent guessed im horrible with naming